### PR TITLE
HPCC-14022 Cannot view logs on multi-node env

### PR DESCRIPTION
**Pull request by miguelvazq**
Whenever navigating to the log viewer within the topology area within ecl watch a "Cannot find logfile.log" message would appear when clicking on the "logs" tab. This request was missing the NetAdress for each request being made therefore throwing this exception.

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review.

File: esp/src/eclwatch/LogWidget.js [hunk 1]
```diff
@@ -231,10 +231,11 @@ define([
         refreshGrid: function (clearSelection) {
             this.rawText.setText(this.i18n.loadingMessage);
             var filter = lang.mixin(this.getFilter(), {
-                Name: this.params.getLogDirectory() + "/" + this.logTargetSelect.get("value"),
+                Name: "//" + this.params.getNetaddress() + this.params.getLogDirectory() + "/" + this.logTargetSelect.get("value"),
                 Type: "tpcomp_log",
                 LoadData: 1
             });
+            console.log(this.params.getNetaddress()+this.params.getLogDirectory());
```
Comment (GordonSmith) [inline esp/src/eclwatch/LogWidget.js]: Remove console.log line.

**Comment by GordonSmith** [comment]
Should remove the debugging line, but otherwise looks fine.

**Comment by miguelvazq** [comment]
Issue with a file. Let me fix.

**Comment by miguelvazq** [comment]
Closing due to file conflicts. https://github.com/hpcc-systems/HPCC-Platform/pull/7651

**Pull request closed without merging**